1.8/usage/haproxy.md

```diff
@@ -0,0 +1,77 @@
+---
+post_title: Configuring HTTP Proxy in Front of DC/OS
```

Comment:
I think we should say "in front of DC/OS Admin Router" instead of just "in front of DC/OS". Then it's maybe clearer how this proxy setup differs from Marathon LB.